cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1057,7 +1057,7 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_palo` sourcetype="pan:system" severity IN ("critical", "high") \
+search = `cs_palo` sourcetype="pan:system" severity IN ("critical", "high") NOT "license for" \
```

Comment:
Add the following to the description of the alert.
"Anything license-related is excluded from this alert as generally it's not that high severity. You can view them on the Palo Alto dashboard."